pkg/api/vlabs/types.go

```diff
@@ -143,6 +143,21 @@ type OrchestratorProfile struct {
 	KubernetesConfig    *KubernetesConfig `json:"kubernetesConfig,omitempty"`
 }
 
+// VersionInfo contains rersion and release numbers
```

Comment:
This file is not the right place for these structs. Can be in upgradetypesandvalidate or in a new file